src/test/groovy/com/stormpath/tck/login/SocialLoginIT.groovy

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2016 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stormpath.tck.login
+
+import com.jayway.restassured.http.ContentType
+import com.jayway.restassured.path.xml.XmlPath
+import com.jayway.restassured.path.xml.element.Node
+import com.jayway.restassured.path.xml.element.NodeChildren
+import com.jayway.restassured.response.Response
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.Iso8601Utils
+import com.stormpath.tck.util.*
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.delete
+import static com.jayway.restassured.RestAssured.given
+import static com.jayway.restassured.RestAssured.put
+import static org.hamcrest.Matchers.*
+import static org.testng.Assert.*
+
+@Test
+class SocialLoginIT extends AbstractIT {
+    // We are really only going to try testing LoginWithFacebook for now, so might have some things hardcoded.
+
+    private String loginRoute = "/login"
+
+    private String facebookClientID
+    private String facebookClientSecret
+
+    private String facebookTestUserAccessToken
+    private String facebookTestUserEmail
+
+    @BeforeClass
+    private void getSocialProviderTokens() throws Exception {
+        getSocialProviderIdsFromStormpath()
```

Comment:
Since the tests in this class depends on `EnvUtils.stormpathApplicationHref`, it wouldn't hurt to assert that it's not empty here.